packages/migrate/src/index.ts

```diff
@@ -29,15 +33,17 @@ const Migrate = {
     if (answer.proceed) {
       return true;
     }
-    await emitEvent(options, "migrate:dryRun:notAccepted");
+    if (options) {
+      await emitEvent(options, "migrate:dryRun:notAccepted");
+    }
```

Comment:
Why `if (options)`?

Reply:
`options` in this method is an optional parameter so we need to make sure it is defined.